styleguide-app/Examples/Balloon.elm

```diff
@@ -103,41 +102,43 @@ paddingOptions =
 
 {-| -}
 type Msg
-    = SetDebugControlsState (Control Settings)
+    = SetCopy (Control String)
+    | SetAttributes (Control (List ( String, Balloon.Attribute )))
 
 
 update : Msg -> State -> ( State, Cmd Msg )
 update msg state =
     case msg of
-        SetDebugControlsState newDebugControlsState ->
-            ( newDebugControlsState
+        SetCopy copy ->
+            ( { state | copy = copy }
+            , Cmd.none
+            )
+
+        SetAttributes attributes ->
+            ( { state | attributes = attributes }
             , Cmd.none
             )
 
 
 view : State -> List (Html Msg)
 view state =
     let
-        settings =
-            Control.currentValue state
+        copy =
+            Control.currentValue state.copy
 
         attributes =
-            List.filterMap identity
-                [ settings.theme
-                , settings.position
-                , settings.width
-                , settings.padding
-                ]
+            Control.currentValue state.attributes
     in
-    [ Control.view SetDebugControlsState state |> fromUnstyled
+    [ Control.view SetCopy state.copy |> fromUnstyled
```

Comment:
elm-css pretty much copies _all_ of `elm/html`, and has its own version of the `elm/html` node types! See https://github.com/rtfeldman/elm-css/blob/17.0.1/src/VirtualDom/Styled.elm#L42 and https://github.com/elm/virtual-dom/blob/master/src/VirtualDom.elm#L40.

We need to use `fromUnstyled` here because `Control.view` produces the `elm/html` `Html` type and the rest of our `Html` is the elm-css version.

It's definitely confusing! There was a period when much of our monolith codebase was a mix of elm-css html and elm html and folks exposed to it for the first time were often very confounded by the whole thing 😖